-1

Tested at: bddbb7e781d8b2c084fe1dc225db34705ab19f83

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14452/12915/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Compiling python modules src/ElectroWeakAnalysis/ZEE/python
>> Compiling python modules src/RecoVertex/AdaptiveVertexFinder/python
>> All python modules compiled
>> Pluging of all type refreshed.
gmake[1]: Leaving directory '/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-05-10-2300'
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 1
</pre>

--- a/DataFormats/EgammaTrackReco/BuildFile.xml
+++ b/DataFormats/EgammaTrackReco/BuildFile.xml
@@ -1,5 +1,6 @@
 <use   name="DataFormats/Common"/>
 <use   name="DataFormats/TrackReco"/>
+<use   name="TrackingTools/PatternTools"/>
 
 <use   name="clhepheader"/>
 <export>

--- a/DataFormats/GeometrySurface/interface/Surface.h
+++ b/DataFormats/GeometrySurface/interface/Surface.h
@@ -39,8 +39,8 @@ using TangentPlane = Plane;
  *  (e.g. Plane or Cylinder).
  */
 
-class Surface : public GloballyPositioned<float> 
-	      , public ReferenceCountedInConditions 
+class Surface : public ReferenceCountedInConditions
+              , public GloballyPositioned<float>
 {
 public:
   using Side =  SurfaceOrientation::Side;

